emperor/support_files/js/sceneplotview3d.js

```diff
@@ -397,7 +397,12 @@ define([
           text = decomp.axesNames[index];
         }
       }
-      text += ' (' + decomp.percExpl[index].toPrecision(4) + ' %)';
+
+      // account for custom axes (their percentage explained will be -1 to
+      // indicate that this attribute is not meaningful).
+      if (decomp.percExpl[index] >= 0) {
```

Comment:
Ah, I wanted to use N/As but then the formatting code in the jinja template would coerce the values to zeros, that's why I ended up using a negative value. There should never be negative percentage of explained variances, that's why I went with this.